diff --git a/node_modules/@fatsolutions/tongo-sdk/dist/provers/audit.js b/node_modules/@fatsolutions/tongo-sdk/dist/provers/audit.js
index bc6a5ed..251e1b3 100644
--- a/node_modules/@fatsolutions/tongo-sdk/dist/provers/audit.js
+++ b/node_modules/@fatsolutions/tongo-sdk/dist/provers/audit.js
@@ -7,7 +7,7 @@ const she_1 = require("@fatsolutions/she");
 const protocols_1 = require("@fatsolutions/she/protocols");
 const constants_1 = require("../constants");
 const types_1 = require("../types");
-const utils_1 = require("../../src/utils");
+const utils_1 = require("../utils");
 // cairo string 'audit'
 exports.AUDIT_CAIRO_STRING = 418581342580n;
 /**
diff --git a/node_modules/@fatsolutions/tongo-sdk/dist/provers/fund.js b/node_modules/@fatsolutions/tongo-sdk/dist/provers/fund.js
index 15dc698..d3099a7 100644
--- a/node_modules/@fatsolutions/tongo-sdk/dist/provers/fund.js
+++ b/node_modules/@fatsolutions/tongo-sdk/dist/provers/fund.js
@@ -6,7 +6,7 @@ exports.verifyFund = verifyFund;
 const she_1 = require("@fatsolutions/she");
 const protocols_1 = require("@fatsolutions/she/protocols");
 const constants_1 = require("../constants");
-const utils_1 = require("../../src/utils");
+const utils_1 = require("../utils");
 const types_1 = require("../types");
 // cairo string 'fund'
 exports.FUND_CAIRO_STRING = 1718972004n;
diff --git a/node_modules/@fatsolutions/tongo-sdk/dist/provers/ragequit.js b/node_modules/@fatsolutions/tongo-sdk/dist/provers/ragequit.js
index 297282d..34cba82 100644
--- a/node_modules/@fatsolutions/tongo-sdk/dist/provers/ragequit.js
+++ b/node_modules/@fatsolutions/tongo-sdk/dist/provers/ragequit.js
@@ -6,7 +6,7 @@ exports.verifyRagequit = verifyRagequit;
 const protocols_1 = require("@fatsolutions/she/protocols");
 const constants_1 = require("../constants");
 const types_1 = require("../types");
-const utils_1 = require("../../src/utils");
+const utils_1 = require("../utils");
 const she_1 = require("@fatsolutions/she");
 // cairo string 'ragequit'
 exports.RAGEQUIT_CAIRO_STRING = 8241982478457596276n;
diff --git a/node_modules/@fatsolutions/tongo-sdk/dist/provers/transfer.js b/node_modules/@fatsolutions/tongo-sdk/dist/provers/transfer.js
index 90a0ed7..799e6e9 100644
--- a/node_modules/@fatsolutions/tongo-sdk/dist/provers/transfer.js
+++ b/node_modules/@fatsolutions/tongo-sdk/dist/provers/transfer.js
@@ -9,7 +9,7 @@ const protocols_2 = require("@fatsolutions/she/protocols");
 const constants_1 = require("../constants");
 const range_1 = require("../provers/range");
 const types_1 = require("../types");
-const utils_1 = require("../../src/utils");
+const utils_1 = require("../utils");
 // cairo string 'transfer'
 exports.TRANSFER_CAIRO_STRING = 8390876182755042674n;
 /**
diff --git a/node_modules/@fatsolutions/tongo-sdk/dist/provers/withdraw.js b/node_modules/@fatsolutions/tongo-sdk/dist/provers/withdraw.js
index 127ee86..f7ee1f2 100644
--- a/node_modules/@fatsolutions/tongo-sdk/dist/provers/withdraw.js
+++ b/node_modules/@fatsolutions/tongo-sdk/dist/provers/withdraw.js
@@ -9,7 +9,7 @@ const protocols_2 = require("@fatsolutions/she/protocols");
 const constants_1 = require("../constants");
 const range_1 = require("../provers/range");
 const types_1 = require("../types");
-const utils_1 = require("../../src/utils");
+const utils_1 = require("../utils");
 // cairo string 'withdraw'
 exports.WITHDRAW_CAIRO_STRING = 8604536554778681719n;
 /**
